mozetl/clientsdaily/fields.py

```diff
@@ -157,6 +163,8 @@ def mode(l, empty_value='MISSING'):
     F.first('search_cohort').alias('search_cohort'),
     F.sum('search_count').alias('search_count_sum'),
     F.mean('session_restored').alias('session_restored_mean'),
+    F.sum(F.expr("IF(subsession_counter = 1, 1, 0)")).alias(
+        "sessions_started_on_this_day"),
```

Comment:
This counts the number of first subsession profiles?

Reply:
It counts the number of sessions started by this client on this day.